classes/cave.js

```diff
@@ -443,6 +446,32 @@ class Cave {
 
                 }
             });
+
+            // let user know that ticket was submitted and give option to remove ticket
+            let removeTicketEmoji = '⚔️';
+
+            let reqTicketUserEmbedMsg = await discordServices.sendEmbedToMember(user, {
+                title: 'Ticket was Succesfull!',
+                description: 'Your ticket to the ' + this.caveOptions.name + ' group was succesful!',
+                fields: [{
```

Comment:
Lets add some ticket information here! We can add it as a field. Ticket description: - description.